tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/DebugAdapterBaseTestCase.java

```diff
@@ -356,15 +360,18 @@ private void setBreakpoints(List<BallerinaTestDebugPoint> breakPoints) throws Ba
                     .stackTrace(stackTraceArgs);
             StackFrame[] stackFrames = stackTraceResp.getStackFrames();
             if (stackFrames.length == 0) {
-                return new Variable[0];
+                return variables;
+            }
+            if (scope == VariableScope.LOCAL) {
```

Comment:
May be we can use the conditional operator here

Reply:
Done via db9e03c